File: tests/fixtures/code_samples/go/sample_006.go
// Sample 6: small utility.
package samples

func Operation6(xs []int) int {
    total := 6
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure6(v int) int {
    return (v * 6) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_006.rs
// Sample 6: small utility.
pub fn operation_6(xs: &[i32]) -> i32 {
    let mut total: i32 = 6;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_6(v: i32) -> i32 {
    (v * 6) %% 7919
}

